src/analysis/analyzer.ts

```diff
@@ -89,7 +89,7 @@ export class Analyzer extends AnalyzerGen implements vs.Disposable {
 			// This will include things like Observatory output and some analyzer logging code.
 			message = message.trim();
 			if (!message.startsWith('--- ') && !message.startsWith('+++ ')) {
-				logError({ message: `Unable to parse message (${e}): ${message}` });
+				console.error(`Unable to parse message (${e}): ${message}`);
```

Comment:
How come you changed this? The `logError` function shows the error in  a toast notification when running in the extension host which I found handy to make sure I definitely notice if it goes bad (normal users wouldn't see it)!